accepted/0020-py-version-support.md

```diff
@@ -29,10 +29,8 @@ for that operating system.
 Including Python in Packages
 ----------------------------
 Another motivating factor to make this policy change is our shift to including Python in our
-packages. For the Sodium release we will be using pop-build (https://gitlab.com/saltstack/pop/pop-build/) to
-package for our Fully Supported Platforms.
-(http://get.saltstack.com/rs/304-PHQ-615/images/SaltStack-Supported-Operating-Systems.pdf)
-This will allow us to easily continue supporting operating systems that include End of Life Python
+packages. For the Sodium release we will be using [Tiamet](https://gitlab.com/saltstack/pop/tiamat/) (previously known as [pop-build](https://gitlab.com/saltstack/pop/pop-build/)) to
```

Comment:
I think it's `Tiamat`

Reply:
@twangboy My word, sorry about that!

Reply:
@twangboy Fixed it, thanks for catching that.